app/lib/admin/actions/user_info.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../../account/agent.dart';
+import '../../account/backend.dart';
+import '../../account/models.dart';
+import '../../package/backend.dart';
+import '../../publisher/backend.dart';
+
+import 'actions.dart';
+
+final userInfo = AdminAction(
+  name: 'user-info',
+  summary:
+      'List info about a user, including all the packages and publishers that a user is an uploader on.',
+  description: '''
+Retrieves info about a user.
+
+Output is on this form:
+```
+{
+  'userId': 'e55b9604-933f-4492-bf55-66cb6c57d7b0',
+  'email': 'admin@pub.dev',
+  'packages': ['flutter_titanium', 'oxygen'],
+  'publishers': ['example.com'],
+  'moderated': false,
+  'created': '2024-04-30T07:32:19.000041Z',
+  'deleted': false
+}
+```
+''',
+  options: {
+    'user': 'The user-id or the email of the user to inspect',
+  },
+  invoke: (options) async {
+    final userIdOrEmail = options['user'];
+    InvalidInputException.check(
+      userIdOrEmail != null && userIdOrEmail.isNotEmpty,
+      'user must be given',
+    );
+
+    User? user;
+    if (looksLikeUserId(userIdOrEmail!)) {
+      user = await accountBackend.lookupUserById(userIdOrEmail);
+    } else {
+      final users = await accountBackend.lookupUsersByEmail(userIdOrEmail);
```

Comment:
It may be useful here to return all the user entries with the given email. Could be in a follow-up later, maybe add a TODO for it?

Reply:
Updated to report on all users with given email.